indico/legacy/webinterface/tpls/ConfDisplayFrame.tpl

```diff
@@ -17,7 +17,7 @@ else:
             <div class="confTitle">
                 <h1>
                     <a href="${ displayURL }">
-                        <span class="conferencetitlelink" ${textColorStyle}>
+                        <span class="conference-title-link" ${textColorStyle}>
```

Comment:
I like the change (since the old name is awful), but we need to update custom stylesheets in the DB in production:

```
>>> Event.query.filter(Event.stylesheet.contains('conferencetitlelink')).count()
339
```